Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1368,6 +1368,11 @@
     <!-- XACT: Statistics explanation illustration description -->
     <string name="statistics_explanation_illustration_description">"Abstrakte Darstellung eines Smartphones mit vier Platzhaltern für Informationen"</string>
 
+    <!-- XTXT: Statistics Card Announcement -->
+    <string name="accessibility_statistics_card_announcement">"Statistikkarte"</string>
+    <!-- XTXT: Statistics Card Navigation Announcement -->
+    <string name="accessibility_statistics_card_navigation_information">"Wische horizontal zwischen den Items für weitere Statistiken"</string>
```

Comment:
maybe "cards or card informations" (Karten) would be a better terminus than items.
But I won't complain. Just a thought

Reply:
Yeah that definitely makes more sense here, wasn't happy with items either (especially in German)